src/Data/Nat/Binary/Minus.agda

```diff
@@ -0,0 +1,364 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Subtraction on Bin and some of its property proofs.
+------------------------------------------------------------------------
+
+{-# OPTIONS --without-K --safe #-}
+
+module Data.Nat.Binary.Minus where
+
+open import Algebra using (Magma)
+import Algebra.FunctionProperties as FuncProp
+open import Data.Empty using (⊥)
+open import Data.Fin using (#_)
+open import Data.Nat as ℕ using (ℕ)
+open import Data.Nat.Binary.Base
+open import Data.Nat.Binary.Properties
+import Data.Nat.Properties as ℕₚ
+open import Data.Product using (_×_; _,_; proj₁; proj₂; ∃)
+open import Data.Sum using (inj₁; inj₂)
+open import Data.Vec using ([]; _∷_)
+open import Function using (_∘_; _$_)
+open import Level using (0ℓ)
+open import Relation.Binary using (Tri; _Preserves_⟶_; _Preserves₂_⟶_⟶_)
+open import Relation.Binary.PropositionalEquality as P using
```

Comment:
At this point I think we can just remove the `using` syntax.

Reply:
Removed.